ChaRo-iOS/ChaRo-iOS/Source/Views/VCs/CreatePostScene/MapScene/AddressMainVC.swift

```diff
@@ -21,50 +21,44 @@ class AddressMainVC: UIViewController {
     private var sendedPostData: WritePostData?
     private var imageList: [UIImage] = []
     
-    private var tableView = UITableView()
-    private var oneCellHeight = 48
-    private var tableViewHeight = 96
-    private var tableViewBottomOffset = 19
+    private lazy var tableView = UITableView().then{
```

Comment:
```suggestion
    private lazy var tableView = UITableView().then {
```
전체적으로 then 뒤에 스페이스가 있으면 좋을 것 같아요~

Reply:
반영했습니다 :) https://github.com/TeamChaRo/ChaRo-iOS/pull/192/commits/879a94e089f395af987f7355296c71fd3219dfa4